README.md

```diff
@@ -51,10 +51,10 @@ It scales nicely from just a single server to thousands of servers, even in comp
   Clear and precise visualization that allows you to quickly understand any dataset, but also to filter, slice and dice the data directly on the dashboard, without the need to learn any query language.
 
 - :bell: **Out of box Alerts**<br/>
-  Comes with hundreds of alerts out of the box to detect common issues and pitfalls, revealing issues that can easily go unnoticed. It supports several notification methods to let you know when your attention is needed.
+  It comes with hundreds of alerts out of the box to detect common issues and pitfalls, revealing issues that can easily go unnoticed. It supports several notification methods to inform you when your attention is needed.
```

Comment:
i think way it was is better given the context of being near top of readme - it's supposed to be chill sort of main quick takeaways 

e.g. in this setting i think "let you know" is better than "inform". And not "Comes" instead of "It comes" is fine as its all ready been established by the "Key Features:" that the subject is netdata, hence no need for "It comes" instead of "Comes"